evm-ext/src/solidity_abi.rs

```diff
@@ -478,7 +478,6 @@ mod tests {
     };
 
     #[test]
-    #[ignore = "flaky"]
     fn test_gas_costs() {
```

Comment:
Yes, it still fails sometimes. I don't think we're there yet.

```suggestion
    #[ignore = "flaky"]
    fn test_gas_costs() {
```

Reply:
Good catch.
What's expected here, that it passes 65% of the time?

Reply:
I have created #425 to address this long term. Leaving the test marked as flaky for now.